app/src/ui/lib/list/section-list.tsx

```diff
@@ -0,0 +1,1680 @@
+import * as React from 'react'
```

Comment:
Random note for others reviewing this. I had this pulled down so I replaced the contents from this into `list.tsx` and reviewed it in Desktop so I could see what was different between this and the `list.tsx`.